site/gdocs/pages/GdocPost.tsx

```diff
@@ -129,7 +138,23 @@ export function GdocPost({
                     className="grid grid-cols-12-full-width col-start-1 col-end-limit"
                 >
                     <div className="col-start-4 span-cols-8 col-md-start-3 span-md-cols-10 col-sm-start-2 span-sm-cols-12">
-                        <h3>Cite this work</h3>
+                        <h3
+                            className={
+                                isDeprecated ? "align-left" : "align-center"
+                            }
+                        >
+                            Cite this work
+                        </h3>
+                        {isDeprecated && (
+                            <p className="citation-deprecated-notice">
+                                <span className="citation-deprecated-notice__highlight">
+                                    This content is outdated
+                                </span>{" "}
+                                but if you would still like to use it, here is
+                                how to cite it.
```

Comment:
I think there should be a comma before the but, and a colon instead of period at the end.

```suggestion
                            <p className="citation-deprecated-notice">
                                <span className="citation-deprecated-notice__highlight">
                                    This content is outdated
                                </span>
                                , but if you would still like to use it, here is
                                how to cite it:
```